tests/Aspire.Microsoft.Data.SqlClient.Tests/ConformanceTests.cs

```diff
@@ -1,23 +1,35 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
+using Aspire.Components.Common.Tests;
 using Aspire.Components.ConformanceTests;
+using Aspire.SqlServer.Tests;
 using Microsoft.Data.SqlClient;
+using Microsoft.DotNet.RemoteExecutor;
 using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Hosting;
+using Xunit;
 
 namespace Aspire.Microsoft.Data.SqlClient.Tests;
 
-public class ConformanceTests : ConformanceTests<SqlConnection, MicrosoftDataSqlClientSettings>
+public class ConformanceTests : ConformanceTests<SqlConnection, MicrosoftDataSqlClientSettings>, IClassFixture<SqlServerContainerFixture>
 {
+    private readonly SqlServerContainerFixture _containerFixture;
+    protected override bool CanConnectToServer => RequiresDockerTheoryAttribute.IsSupported;
+    private string ConnectionString => RequiresDockerTheoryAttribute.IsSupported
+                                        ? _containerFixture.GetConnectionString()
+                                        : "Data Source=fake;Database=master";
     protected override ServiceLifetime ServiceLifetime => ServiceLifetime.Scoped;
 
     // https://github.com/open-telemetry/opentelemetry-dotnet/blob/031ed48714e16ba4a5b099b6e14647994a0b9c1b/src/OpenTelemetry.Instrumentation.SqlClient/Implementation/SqlActivitySourceHelper.cs#L31
     protected override string ActivitySourceName => "OpenTelemetry.Instrumentation.SqlClient";
 
-    // TODO
-    protected override string[] RequiredLogCategories => Array.Empty<string>();
+    protected override string[] RequiredLogCategories => [
+        "Microsoft.Extensions.Hosting.Internal.ApplicationLifetime",
```

Comment:
Were tests failing if we don't do this? We don't have these categories elsewhere.

The issue is that Microsoft.Data.SqlClient doesn't support logging. See `Aspire.Microsoft.Data.SqlClient:` in https://github.com/dotnet/aspire/blob/main/src/Components/Telemetry.md.

Reply:
No, the tests weren't failing because they didn't check for any required categories.

From your link:
```
Aspire.Microsoft.EntityFrameworkCore.SqlServer:

Log categories:
"Microsoft.EntityFrameworkCore.ChangeTracking"
"Microsoft.EntityFrameworkCore.Database.Command"
"Microsoft.EntityFrameworkCore.Database.Connection"
"Microsoft.EntityFrameworkCore.Database.Transaction"
"Microsoft.EntityFrameworkCore.Infrastructure"
"Microsoft.EntityFrameworkCore.Migrations"
"Microsoft.EntityFrameworkCore.Model"
"Microsoft.EntityFrameworkCore.Model.Validation"
"Microsoft.EntityFrameworkCore.Query"
"Microsoft.EntityFrameworkCore.Update"
```

Shouldn't we check for these?

Reply:
oops, you were talking about `SqlClient`:
```
Aspire.Microsoft.Data.SqlClient:
Log categories:
none (the client does not provide an easy way to integrate it with logger factory)
```